dashboards/node-overview.json

```diff
@@ -85,6 +85,23 @@
         }
       }
     },
+    {
+      "title": "sysproc cpu utilization (Trinity+)",
+      "datasource": "{data-source-name}",
+      "_base": "panel",
+      "_targets": [
+        {
+          "expr": "(rate(sysproc_cpu_user[1m]) + ignoring(name) rate(sysproc_cpu_sys[1m])) / ignoring(category, name, proc) group_left sys_cpu_cores_available * 100",
```

Comment:
Should this be irate?

Reply:
I wasn't sure how to define slow moving counters...based on this description I had found

rate() is generally used when graphing the slow moving counters. While irate() is used when graphing the high volatile counters.

Reply:
CPU isn't a "slow moving counter" though. It's the textbook definition of a volatile counter. CPU can spike from 0->100 in seconds (even milliseconds) - and we don't want to average this and show it slowly going from 0 to 100 over the course of a minute.

Reply:
I agree, it should be irate